src/Serializer/DoctrineDenormalizer.php

```diff
@@ -30,11 +37,16 @@ final class DoctrineDenormalizer implements DenormalizerInterface
      *
      * @param \Doctrine\Common\Persistence\ManagerRegistry $managerRegistry
      * @param mixed[]|null $classKeyMap
+     * @param mixed[]|null $ignoreClasses
      */
-    public function __construct(ManagerRegistry $managerRegistry, ?array $classKeyMap = null)
-    {
+    public function __construct(
+        ManagerRegistry $managerRegistry,
+        ?array $classKeyMap = null,
+        ?array $ignoreClasses = null
+    ) {
         $this->managerRegistry = $managerRegistry;
         $this->classKeyMap = $classKeyMap;
+        $this->ignoreClasses = $ignoreClasses;
```

Comment:
You can just do ?? []; to avoid the is_array check below

Reply:
done